subworkflows/local/align_MT.nf

```diff
@@ -51,20 +51,22 @@ workflow ALIGN_MT {
         ch2=SMATOOLS_SORT_MT.out.bam.join(SMATOOLS_INDEX_MT.out.bai, by: [0])
         ch3=ch2.combine(intervals_mt)
         ch_versions = ch_versions.mix(SMATOOLS_INDEX_MT.out.versions.first())
-     
+
         // Calls variants with Mutect2
         GATK4_MUTECT2_MT ( ch3, fasta, fai, dict, [], [], [], [] )
         ch_versions = ch_versions.mix(GATK4_MUTECT2_MT.out.versions.first())
 
         // Haplocheck
-     //   HAPLOCHECK_MT ( GATK4_MUTECT2_MT.out.vcf )
-     //   ch_versions = ch_versions.mix(HAPLOCHECK_MT.out.versions.first())
+        // TODO: probably it will be outside this subworkflow as we want to run
+        // with the VCF with the variants from the shifted alignment (to solve the mt circularity issue)
```

Comment:
Don't we want to run Haplocheck for both the shifted and unshifted alignments?

Reply:
I'm also not sure, but I think we should provide _haplocheck_ with the whole variant set - obtained by merging standard and lifted ones - because it will use each polymorphic site to estimate contamination and haplogroup.